posts/2021-10-05-openliberty-jug-presentation.adoc

```diff
@@ -14,18 +14,21 @@ additional_authors:
   github: 
   image:
 ---
-= Why Open Liberty is Your Next Cloud Native Java Runtime
+= A Fun Cup of Joe: Find Out Why Open Liberty is Your Next Cloud Native Java Runtime
 Andrew Mauer <https://github.com/ajm01>
 :imagesdir: /
 :url-prefix:
 :url-about: /
 //Blank line here is necessary before starting the body of the post.
 
+== Why Open Liberty?
 As the popularity of a microservice architecture and cloud-native application development experience grows, it is an exciting time to talk about the Open Liberty framework's support of those. I recently had the opportunity to lead a discussion at a meeting of the Philadelphia Java User Group (JUG) meetup where I highlighted the latest features and design ideas contained within Open Liberty. As part of that discussion I ran two demos where I created and interacted with an Open Liberty-hosted RESTful web service as an application developer - both in a local developer environment and live in a remote Kubernetes cluster environment. 
 
-The focus of the presentation centered around Open Liberty as an open framework for Java developers that highly optimizes the development of microservices using Eclipse MicroProfile and Jakarta EE APIs. To highlight Open Liberty as a flexible, fast and lightweight Java runtime, the discussion touched on important Open Liberty framework design attributes such as its simple configuration to enable "Just Enough Runtime" - configuring the framework to enable only the subset of features an application needs at runtime, its low operating cost due to a reduced memory footprint based on that minimal configuration, zero migration, Kubernetes optimizations and its efficient rapid developer experience. Additionally, the fact that Open Liberty supports the deployment of Spring Boot applications generated an enthusiastic question from a member of the audience interested in understanding how Open Liberty could help a shop with hundreds of Spring Boot applications become more efficient.
+image::img/blog/cupofjoe.JPG[A Fun Cup Of Joe, align="center"]
```

Comment:
Not entirely sure, but I think images are supposed to be in png format but I'll let David comment on that when he does his editorial review.